code/modules/surgery/surgery_step.dm

```diff
@@ -90,8 +90,9 @@
 
 	// Skyrat Edit Addition - reward for doing surgery in surgery
 	if(was_sleeping || HAS_TRAIT(target, TRAIT_NUMBED) || target.stat == DEAD)
-		modded_time *= SURGERY_SPEEDUP_AREA
-		to_chat(user, span_notice("You are able to work faster due to the patient's calm attitude!"))
+		if(is_type_in_list(get_area(target), list(/area/medical/surgery, /area/science/robotics)) && (TRAIT_FASTMED in user.status_traits) || (TRAIT_QUICK_CARRY in user.status_traits))
```

Comment:
At a quick glance, this logic doesn'tseem to work, if you have the quick carry trait, you'll be able to do surgery mega fast on people everywhere.

Reply:
i made this super super fast, if you have any idea how to actually make this work, go wild